geopandas/testing.py

```diff
@@ -90,7 +90,7 @@ def _geom_almost_equals_mask(this, that):
     """
 
     return (
-        this.geom_almost_equals(that)
+        this.geom_equals_exact(that)
```

Comment:
This doesn't work as is because `geom_equals_exact` requires tolerance be specified. For this use we should directly translate the equivalent of the old `geom_almost_equals`:
```suggestion
        this.geom_equals_exact(that, tolerance=0.5 * 10 ** (-6))
```